querybook/webapp/components/DataDoc/DataDoc.tsx

```diff
@@ -382,6 +386,46 @@ class DataDocComponent extends React.PureComponent<IProps, IState> {
         toast('Copied.' + (cut ? ' Cell will be moved after paste. ' : ''));
     }
 
+    @bind
+    public deleteCellAt(index: number) {
+        const { dataDoc, docId } = this.props;
+
+        const numberOfCells = dataDoc.dataDocCells.length;
+        const cell = dataDoc.dataDocCells[index];
```

Comment:
is it possible that index is out of range and make `cell` be undefined?

Reply:
shouldn't happen, the focusedIndex can't be cell out of index. If the delete is issued by the cell directly, it should always be valid